lib/pdf/external_font.ex

```diff
@@ -79,6 +110,53 @@ defmodule Pdf.ExternalFont do
     byte_size(to_iolist(font) |> Enum.join())
   end
 
+  @doc """
+  Returns the width of the specific character
+
+  Examples:
+
+    iex> ExternalFont.width(font, "A")
+    123
+  """
+  def width(font, <<char_code::utf8>> = str) when is_binary(str) do
+    width(font, char_code)
+  end
+
+  def width(font, char_code) do
+    Pdf.Encoding.WinAnsi.characters()
+    |> Enum.find(fn {_, char, _} -> char == char_code end)
+    |> case do
+      nil ->
+        0
+
+      {_, _, name} ->
+        case font.glyphs[name] do
+          nil ->
+            0
+
+          %{width: width} ->
+            width
+        end
+    end
+  end
+
+  def kern_text(_font, ""), do: [""]
+
+  def kern_text(font, <<first::utf8, second::utf8, rest::binary>>) do
```

Comment:
All `::utf8` needs to be changed to `::integer` to deal with encoding correctly.